src/wallet/coinselection.cpp

```diff
@@ -174,6 +174,10 @@ util::Result<SelectionResult> SelectCoinsBnB(std::vector<OutputGroup>& utxo_pool
                 curr_selection_weight += utxo.m_weight;
             }
         }
+        if (curr_try >= TOTAL_TRIES - 1) {
+            // On last attempt and didn’t break due to full traversal: solution is non-optimal
+            result.SetAlgoCompleted(false);
```

Comment:
In cf14a72859a2eaf42a24a2ea3125bdadde5f6611: Here we're setting it for BnB but seems we're not using `GetAlgoCompleted` for it?

Reply:
You are right, I’ll have follow up on this